lib/lufa_config.h

```diff
@@ -11,7 +11,7 @@
 #define NO_INTERNAL_SERIAL
 #define NO_DEVICE_SELF_POWER
 #define NO_DEVICE_REMOTE_WAKEUP
-// #define NO_SOF_EVENTS
+#define NO_SOF_EVENTS
```

Comment:
:thinking: this sounds dangerous, we had tons of trouble with SOF events on Buddy platform...

Reply:
SOF is not used by the CDC class. In device mode, only stuff that does isochronous streams of data cares about SOF, such as UAC and UVC. So disabling the handling of the flag in the interrupt has no adverse effect. We don't care about the SOF callback.
On the other hand, SOF is generated by the host for any class and is part of the frame structure of USB. Maybe this is what you are remembering?